mbtiles/src/mbtiles.rs

```diff
@@ -317,6 +317,49 @@ impl Mbtiles {
         Ok(())
     }
 
+    /// Check if a tile exists in the database.
+    ///
+    /// This is a slight optimisation over [`Mbtiles::get_tile_and_hash`] or [`Mbtiles::get_tile`] and does only look up IF the tile exists, not the tile data.
+    ///
+    /// <div class="warning">
+    ///
+    /// **Note:**
+    /// You usually (unless the second query is unnecessary) want to retrieve the tile data if it exists instead.
+    /// Sqlite already checks for a tiles existence and retrieving it if it exists in [`Mbtiles::get_tile_and_hash`] or [`Mbtiles::get_tile`].
+    ///
+    /// </div>
+    ///
+    /// *Tipp:*
+    /// If you need even more performance in this specific operation, consider scanning over the entire table once via [`Mbtiles::stream_coords`] and building a bloom filter (f.ex. [`fastbloom`](https://docs.rs/fastbloom/)) as a fast path.
+    /// This only works if
+    /// - the database is static or
+    /// - you maintain the bloom filter on inserts and deletes to the database.
```

Comment:
The simpler the better. I don't think we need to go into the bloom filter here, regardless of how cool that thing is.  I.e. less is more approach :)

```suggestion
    /// Check if a tile exists in the database.
    ///
    /// This method is slightly faster than [`Mbtiles::get_tile_and_hash`] and [`Mbtiles::get_tile`]
    /// because it only checks if the tile exists but does not retrieve tile data.
    /// Most of the time you would want to use the other two functions.
```

Reply:
P.S. similar change for the other new method